packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/home/home.controller.js

```diff
@@ -0,0 +1,21 @@
+import { ITEMS_POLL_INTERVAL } from '../../ai-dashboard.constants';
+
+export default class AIDashboardHomeCtrl {
+  /* @ngInject */
+  constructor(atInternet, $translate, AIDashboardService) {
+    this.atInternet = atInternet;
+    this.$translate = $translate;
+    this.AIDashboardService = AIDashboardService;
+  }
+
+  $onInit() {
+    // Poll data
+    setInterval(() => this.getAIItems(), ITEMS_POLL_INTERVAL);
```

Comment:
**perf:** Clear interval in `$destroy`

Reply:
I have removed the file, as devs for this tab are on another PR (where the interval is cleared: https://github.com/ovh/manager/pull/8172/files#diff-07e84ffe020e64e0668d659252a944e45cac7c581d957ad25f975f65f25d4e2dR39-R41)